charts/kafka-operator/crds/operator-kafka-crd.yaml

```diff
@@ -514,6 +519,11 @@ spec:
                         additionalProperties:
                           type: string
                         type: object
+                        description: 'Custom annotations for the broker pods:
```

Comment:
can we actually add this description in types file so kubebuilder automatically generates this field description?